functional_testing/0_orchestrator/README.md

```diff
@@ -0,0 +1,30 @@
+## 0-Orchestrator Testsuite
```

Comment:
Please remove this read me file .. it's already integrated in the orchestrator repo